pandas/tests/test_base.py

```diff
@@ -1269,3 +1269,54 @@ def test_ndarray_values(array, expected):
     r_values = pd.Index(array)._ndarray_values
     tm.assert_numpy_array_equal(l_values, r_values)
     tm.assert_numpy_array_equal(l_values, expected)
+
+
+@pytest.mark.parametrize("array, attr", [
```

Comment:
maybe put in pandas/tests/arrays/test_arrays.py?

Reply:
Doesn't exist yet :), though my `pd.array` PR is creating it.

This seemed a bit more appropriate since it's next to our tests for `ndarray_values`.